Anytype/Sources/PresentationLayer/TextEditor/Set/Views/Popups/ViewPicker/SetViewPicker.swift

```diff
@@ -53,7 +56,8 @@ struct SetViewPicker: View {
         .toolbar {
             ToolbarItem(placement: .navigationBarLeading) {
                 EditButton()
-                    .foregroundColor(Color.Button.active)
+                    .buttonDynamicForegroundColor()
```

Comment:
In relations we just hide buttons in toolbar in non-editable mode, maybe we should make it here in the same way?

Reply:
fixed